<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSecurity</class>
 <widget class="QMainWindow" name="WidgetSecurity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Security</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Resource.qrc">
    <normaloff>:/Resource/Security/Security.png</normaloff>:/Resource/Security/Security.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTableView" name="tableViewSecurity"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBarControls">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSecurityAddRule"/>
   <addaction name="actionSecurityRemoveRule"/>
   <addaction name="actionSecurityModifyRule"/>
   <addaction name="actionSecurityImportRules"/>
   <addaction name="actionSecurityExportRules"/>
  </widget>
  <widget class="QToolBar" name="toolBarSubscribe">
   <property name="contextMenuPolicy">
    <enum>Qt::PreventContextMenu</enum>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSubscribeSecurityList"/>
  </widget>
  <action name="actionSecurityAddRule">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Security/AddRule.png</normaloff>:/Resource/Security/AddRule.png</iconset>
   </property>
   <property name="text">
    <string>Add Rule</string>
   </property>
   <property name="toolTip">
    <string>Add Rule</string>
   </property>
  </action>
  <action name="actionSecurityRemoveRule">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Security/RemoveRule.png</normaloff>:/Resource/Security/RemoveRule.png</iconset>
   </property>
   <property name="text">
    <string>Remove Rule</string>
   </property>
   <property name="toolTip">
    <string>Remove Rule</string>
   </property>
  </action>
  <action name="actionSecurityModifyRule">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Security/ModifyRule.png</normaloff>:/Resource/Security/ModifyRule.png</iconset>
   </property>
   <property name="text">
    <string>Modify Rule</string>
   </property>
   <property name="toolTip">
    <string>Modify Rule</string>
   </property>
  </action>
  <action name="actionSecurityImportRules">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Security/ImportRules.png</normaloff>:/Resource/Security/ImportRules.png</iconset>
   </property>
   <property name="text">
    <string>Import Rules</string>
   </property>
   <property name="toolTip">
    <string>Import Rules</string>
   </property>
  </action>
  <action name="actionSecurityExportRules">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Security/ExportRules.png</normaloff>:/Resource/Security/ExportRules.png</iconset>
   </property>
   <property name="text">
    <string>Export Rules</string>
   </property>
   <property name="toolTip">
    <string>Export Rules</string>
   </property>
  </action>
  <action name="actionSubscribeSecurityList">
   <property name="icon">
    <iconset resource="../Resource.qrc">
     <normaloff>:/Resource/Security/SubscribeRules.png</normaloff>:/Resource/Security/SubscribeRules.png</iconset>
   </property>
   <property name="text">
    <string>Subscribe to Security List</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
</ui>
